src/modules/account/components/notifications/notifications.tsx

```diff
@@ -29,16 +33,20 @@ class Notifications extends React.Component<NotificationsProps> {
   componentWillReceiveProps(nextProps: NotificationsProps) {
     if (!isEqual(nextProps.resolvedMarketsOpenOrders, this.props.resolvedMarketsOpenOrders) ||
         !isEqual(nextProps.reportOnMarkets, this.props.reportOnMarkets) ||
+        !isEqual(nextProps.marketsInDispute, this.props.marketsInDispute) ||
+        !isEqual(nextProps.completeSetPositions, this.props.completeSetPositions) ||
         !isEqual(nextProps.finalizedMarkets, this.props.finalizedMarkets)) {
           this.updateState(nextProps);
     }
   }
 
   updateState(props: NotificationsProps) {
     this.props.updateNotifications(
-      this.generateCards(this.props.resolvedMarketsOpenOrders, 'resolvedMarketsOpenOrders')
-      .concat(this.generateCards(this.props.reportOnMarkets, 'reportOnMarkets'))
-      .concat(this.generateCards(this.props.finalizedMarkets, 'finalizedMarkets')));
+      this.generateCards(props.resolvedMarketsOpenOrders, 'resolvedMarketsOpenOrders')
```

Comment:
'resolvedMarketsOpenOrders' like these can be put in constants as NOTIFICATION_TYPES: {....}

I'd like to remove magic strings.

Reply:
👍 on it